doc/man1/ts.pod

```diff
@@ -288,7 +288,7 @@ to the output file. This option does not require a request, it is
 useful e.g. when you need to examine the content of a response or
 token or you want to extract the time stamp token from a response. If
 the input is a token and the output is a time stamp response a default
-'granted' status info is added to the token. (Optional)
+"granted" status info is added to the token. (Optional)
```

Comment:
Any special reason for this change? If you really want this change, maybe you should change *all* instances of single quotes (watch out do you don't change an apostrophe)

Reply:
In roff, a single quote at the start of a line is just like a period -- it's a macro invocation -- however pod2man does escape it properly. This was a holdover from the previous PR, and I will update the commit to remove it.